adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/routes/AssetsResource.kt

```diff
@@ -48,9 +46,7 @@ import javax.ws.rs.core.Response
 @Tag(name = "Assets")
 @Path("/v3/assets/")
 @Produces(MediaType.APPLICATION_JSON)
-@Timed
 @ApplicationScoped
-@GZIP
```

Comment:
There's no GZIP support in RESTEasy reactive yet, issue opened: https://github.com/quarkusio/quarkus/issues/16425